testing/ostest/sporadic2.c

```diff
@@ -310,9 +310,9 @@ int sporadic2_test(void)
   int32_t budget_2_ns = 30 * 1000000;
   int i;
 
-  printf("Sporadic 1: prio high %d, low %d, repl %" PRIi32 " ns\n",
+  printf("Sporadic 1: prio high %d, low %d, repl %ld ns\n",
```

Comment:
How about changing the definition of REPL_INTERVAL to:
```patch
--- a/testing/ostest/sporadic2.c
+++ b/testing/ostest/sporadic2.c
@@ -44,7 +44,7 @@
 #define PRIO_MID      100
 #define PRIO_HIGH1    180
 #define PRIO_HIGH2    170
-#define REPL_INTERVAL 100000000L
+#define REPL_INTERVAL INT32_C(100000000)
 #define MAX_BUDGET    (REPL_INTERVAL / 2)
 #define PRIO_HI_NDX   0
 #define PRIO_LO_NDX   1
 ```
Then we may keep the format specifier as it currently is.

Reply:
Done.